src/models/methods/subscribe.ts

```diff
@@ -25,9 +25,7 @@ export interface SubscribeRequest extends BaseRequest {
 }
 
 export interface SubscribeResponse extends BaseResponse {
-  // TODO: figure out if there's a better way to type this
-  // eslint-disable-next-line @typescript-eslint/ban-types -- actually should be an empty object
-  result: {} | Stream
+  result: Record<string, never> | Stream
```

Comment:
Just to make sure this doesn't get lost - This field should also be updated with whatever you decide in response to Mayukha's comment above